documentation/user_docs/docs/manual/Cutting_data_of_interest_from_SQW_files_and_objects.rst

```diff
@@ -976,13 +970,256 @@ the spin-wave centre:
    centre at :math:`[0,-1,1]`.
 
 
+.. _cylindrical_projection:
+
 Cylindrical Projections
-^^^^^^^^^^^^^^^^^^^^^^^
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+In order to construct a cylindrical projection (i.e. a projection in :math:`Q_{tr}`, :math:`Q_{||}`,
+:math:`\phi` (azimuthal angle) and :math:`E`) coordinate system we create a projection in a way
+similar to the ``line_proj``, but using the ``cylinder_proj`` class.
+
+The complete signature for ``cylinder_proj`` is similar to ``sphere_proj``:
+
+.. code-block:: matlab
+
+   proj = cylinder_proj([u][, v][, type][, alatt][, angdeg][, offset][, label][, title][, lab1][, lab2][, lab3][, lab4])
+
+where:
+
+- ``u`` The direction of the :math:`z`-axis of the cylindrical coordinate system. 
+    The vector :math:`u` is the reciprocal space vector defining the direction of the :math:`z`-axis 
+    of the target cylindrical coordinate system. See :ref:`fig_cylinder_coodinates` for details.
+
+- ``v``  The direction of the :math:`x`-axis of the cylindrical coordinate system. 
```

Comment:
Again, it might be better to define this as a radial direction forming the basis for $\theta$

Reply:
do not know, looks ok to me.